src/main/java/mate/academy/springbootintro/controller/ShoppingCartController.java

```diff
@@ -64,28 +65,27 @@ public ShoppingCartDto addItemToCart(
             summary = "Update quantity",
             description = "Updating products quantity in item carts"
     )
-    public ShoppingCartDto updateQuantity(
-            @PathVariable @Positive Long cartItemId,
+    public CartItemDto updateQuantity(
```

Comment:
```suggestion
    public ShoppingCartDto updateQuantity(
```

revert changes

Why did you decide to change return type?